### ISSUE TITLE
Quickly create Moments (formerly Triggers) from the Home/Latest page

### ISSUE THREAD

**Issue by julianguyen** [julianguyen/ifme#114]
Be able to quickly create Moments from the Home/Latest page (user must be signed in).

Form includes
* name
* description
* most frequently used categories and moods to select from

**Comment by julianguyen** [comment]
@markfarrell Hey! Wanna check out the live app and let me know what you think about quickly adding Moments (renamed from Triggers)? Thanks :)

**Comment by markfarrell** [comment]
Hey, sure. I like the idea of these changes overall. However, there were a few things I noticed after trying to use these new changes. Here would be current feedback:

- I personally like the rename from moments to triggers; I think it resolves the present tense vs. past tense unintuitiveness that I felt like was slightly there before.
- I noticed that the feature that lets me quickly add moments only appears after I add at least one moment. This might be intentional, but I found this a bit confusing at first. Is there away to (comfortably) have both the if me guidelines and latest moments on the same page?
- I noticed that if I try to add a moment quickly, with a name, but no description of what happened, then it won't add the moment. Is this intentional? Should I receive some sort of message stating I must enter a description of what happened if this is the case?
- In the moments section, I noticed that when I tried to add a new mood while working on a moment, it would erase the moment I was working on. This tripped me up a bit, I felt.
